gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -419,6 +420,10 @@
   public static final HadoopConfigurationProperty<MetricsSink> GCS_METRICS_SINK =
       new HadoopConfigurationProperty<>("fs.gs.metrics.sink", MetricsSink.NONE);
 
+  /** Configuration key to enable publishing event logs to Google cloud logging. */
+  public static final HadoopConfigurationProperty<EventLogSink> GCS_EVENT_LOG_SINK =
+      new HadoopConfigurationProperty<>("fs.gs.eventlog.sink", EventLogSink.NONE);
```

Comment:
nit: `event-log`

Reply:
I am not particularly a fan of introducing `-` in flags/configurations.